ffxiv_visland/Gathering/GatherRouteDB.cs

```diff
@@ -67,6 +78,8 @@ public class Waypoint
         public int QuestID;
         public int QuestSeq;
         public int MobID;
+        public string NodeScanTarget = "";
```

Comment:
Why would you not use InteractWithName?

Reply:
Great point, it hadn't occurred to me to reuse the properties for other waypoint types